### PR TITLE
Add Lukso to command line table

### LINKED ISSUE #1637: Add LUKSO network support
https://github.com/hyperledger/besu/pull/7223

Reference this above pull request. Can also mirror Teku's docs: https://docs.teku.consensys.io/reference/cli#network

### PR DESCRIPTION
**Pull request by joaniefromtheblock**
## Description

-  Add Lukso support

### Issue(s) fixed

Fixes #1637

### CODE REVIEW

**Review by macfarla — APPROVED** [review]

**Review by alexandratran — APPROVED** [review]

File: docs/public-networks/reference/cli/options.md [hunk 1]
```diff
@@ -2636,16 +2636,17 @@ network="holesky"
...
-| `mordor ` | ETC   | Test        | [FAST](#sync-mode) | A PoW network                                                  |
+Possible values include the following:
+
+| Network   | Chain | Type        | Default Sync Mode  | Consensus Mechanism      | Description                                                                          |
+| :-------- | :---- | :-----------| :----------------- | :----------------------- | :----------------------------------------------------------------------------------- |
+| `mainnet` | ETH   | Production  | [FAST](#sync-mode) | A PoS network            | The main [Ethereum network](https://ethereum.org/en/developers/docs/networks/)       |
+| `holesky` | ETH   | Test        | [FAST](#sync-mode) | A PoS network            | Multi-client testnet [Hoelsky](https://holesky.dev)                                  |
+| `sepolia` | ETH   | Test        | [FAST](#sync-mode) | A PoS network            | Multi-client testnet [Sepolia](https://sepolia.dev)                                  |
+| `lukso`   | ETH   | Production  | [FAST](#sync-mode) | A PoS network            | Network for the [Lukso chain](https://lukso.network/)                                |
```
Comment (alexandratran) [inline docs/public-networks/reference/cli/options.md:2646]:
```suggestion
| `lukso`   | ETH   | Test  | [FAST](#sync-mode) | A PoS network            | Network for the [Lukso chain](https://lukso.network/)                                |
```

Comment (joaniefromtheblock) [reply docs/public-networks/reference/cli/options.md:2646]: @alexandratran Sorry this is production according to the [Teku docs](https://docs.teku.consensys.io/reference/cli#network)

File: docs/public-networks/reference/cli/options.md [hunk 2]
```diff
@@ -2636,16 +2636,17 @@ network="holesky"
...
-| :-------- | :---- | :-----------| :----------------- | :------------------------------------------------------------- |
-| `mainnet` | ETH   | Production  | [FAST](#sync-mode) | The main network                                               |
-| `holesky` | ETH   | Test        | [FAST](#sync-mode) | A PoS network                                                  |
-| `sepolia` | ETH   | Test        | [FAST](#sync-mode) | A PoS network                                                  |
-| `dev`     | ETH   | Development | [FULL](#sync-mode) | A PoW network with a low difficulty to enable local CPU mining |
-| `classic` | ETC   | Production  | [FAST](#sync-mode) | The main Ethereum Classic network                              |
-| `mordor ` | ETC   | Test        | [FAST](#sync-mode) | A PoW network                                                  |
+Possible values include the following:
+
+| Network   | Chain | Type        | Default Sync Mode  | Consensus Mechanism      | Description                                                                          |
```
Comment (macfarla) [inline docs/public-networks/reference/cli/options.md:2641]: I think the default sync mode will be SNAP now for everything except `dev`

Comment (macfarla) [reply docs/public-networks/reference/cli/options.md:2641]: but you could do that as a separate PR

**Review by macfarla — APPROVED** [review]
I think the default sync mode will be SNAP now for everything except `dev` but you could do that as a separate PR

**Pull request merged**